src/client/components/ShoppingCart/ShoppingCart.component.js

```diff
@@ -7,32 +7,97 @@ import RemoveFromShoppingCart from './RemoveFromShoppingCart';
 
 export default function ShoppingCart(props) {
   const { shoppingCart, setShoppingCart } = props;
+  const [totalPrice, setTotalPrice] = useState(totalPriceOfProducts);
+  // TODO :add functionality of changing price with quantity
+  // const [totalDiscount,setTotalDiscount]=useState();
+  // const [totalPayment,setTotalPayment]=useState();
+
+  const totalPriceOfProducts = shoppingCart.reduce(
+    (sum, product) => sum + product.quantity * product.price,
+    0,
+  );
+  console.log(totalPriceOfProducts);
+  const totalDiscount = shoppingCart.reduce(
+    (sum, product) =>
+      sum +
+      Math.round((product.discount * (product.quantity * product.price)) / 100),
+    0,
+  );
+  const amountToBePayed = totalPriceOfProducts - totalDiscount;
 
   return (
-    <div>
-      <ul className="cart-container">
-        {shoppingCart.map((product) => (
-          <div>
-            <li key={product.id} className="cart-list"> 
-              <div className="cart-product-container">
-                <ShoppingCartImage product={product} />
-                <ModifyProductQuantity
-                  product={product}
-                  shoppingCart={shoppingCart}
-                  setShoppingCart={setShoppingCart}
-                />
-                <RemoveFromShoppingCart
-                  product={product}
-                  shoppingCart={shoppingCart}
-                  setShoppingCart={setShoppingCart}
-                />
+    <div className="parent-div">
+      1.
```

Comment:
we don't need these numbers!

Reply:
this PR was not ready to review but no worries now you can review again.
thanks